unittests/test_invokust.py

```diff
@@ -1,16 +1,21 @@
-import invokust
+import os
+import sys
 
-settings = invokust.create_settings(
-    locustfile='../locustfile.py',
+from invokust import create_settings, LocustLoadTest
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+settings = create_settings(
+    locustfile='locustfile.py',
     host='https://dss.dev.data.humancellatlas.org/v1/',
```

Comment:
this var is available in Environment file.

Reply:
Thanks, added!